package.json

```diff
@@ -2,7 +2,7 @@
   "name": "markdown-it-attrs",
   "version": "1.0.0",
   "description": "Add classes, identifiers and attributes to your markdown with {} curly brackets, similar to pandoc's header attributes",
-  "main": "index.js",
+  "main": "markdown-it-attrs.browser.js",
```

Comment:
I don't believe it does.

I created a small project that had just `index.js` containing this:
```
var MdIt = require('markdown-it')
var mdAttrs = require('markdown-it-attrs')

var md = MdIt()

md.use(mdAttrs)

var text = 'test __some__ *markdown*{data-toggle=test} with attrs'

var output = md.render(text)
console.log('rendered text = ', output)
```
I got the same output when I ran the packaged v1.0.0 and my updated version.  Both produced the following with no errors:
```
rendered text =  <p>test <strong>some</strong> <em data-toggle="test">markdown</em> with attrs</p>
```
I did `node ./index.js` for both node v6.9.1 and node v8.2.1